src/main/java/filter/expression/Qualifier.java

```diff
@@ -398,6 +399,15 @@ private static boolean isMetaQualifier(Qualifier q) {
             case "date":
                 comparator = (a, b) -> a.getUpdatedAt().compareTo(b.getUpdatedAt());
                 break;
+            case "nonselfupdate":
+                // Purposefully inverted a and b to make latest issues show up first.
+                if (metadataRefresh) {
+                    comparator = (a, b) ->
+                        b.getMetadata().getNonSelfUpdatedAt().compareTo(a.getMetadata().getNonSelfUpdatedAt());
+                } else {
+                    comparator = (a, b) -> b.getUpdatedAt().compareTo(a.getUpdatedAt());
+                }
+                break;
```

Comment:
Oh, so it was intentional to add it as syntax? My bad then.

Do add tests for it. :p